src/transformers/modeling_utils.py

```diff
@@ -2126,6 +2115,26 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                     "Using `low_cpu_mem_usage=True` or a `device_map` requires Accelerate: `pip install accelerate`"
                 )
 
+        if load_in_8bit and quantization_config is not None:
```

Comment:
I don't think this warrants a ValueError here.

Reply:
Should be fixed in 2f281ac